content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,127 @@
+
+2. Alice then takes the cross-chain claim ID from sAlice’s XChainCreateClaimID transaction and submits a XChainCommit transaction on the mainchain with that claim ID, locking up a specified amount of XRP. She specifies sAlice’s account in the OtherChainDestination field.
+
+3. The witnesses then take note of the XChainCommit transaction and submit XChainAddAttestation transactions on the sidechain, attesting to the fact that the XChainCommit transaction did in fact occur on the mainchain.
+
+4. When there are enough XChainAddAttestation signatures to reach quorum, the XRP is automatically released on the sidechain to sAlice’s account.
+
+5. If the XRP is not automatically released, for whatever reason (such as Alice forgetting to specify sAlice’s account in the OtherChainDestination field), then sAlice submits a XChainClaim transaction on the sidechain, specifying her account as the destination. This then releases the XRP on the sidechain to sAlice’s account.
+
+
+## Terminology
+
+* Bridge: A method of moving assets/value from one blockchain to another.
+
+* Witness: Independent servers that are aware of the locking and issuing chains. See [witness servers](witness-server.md) for more information.
+
+* Cross-chain transfer: A transfer of assets from one chain to another.
+
+* Source chain: The chain that a cross-chain transfer begins from. The transfer is from the source chain and to the destination chain.
+
+* Destination chain: The chain that a cross-chain transfer ends at. The transfer is from the source chain and to the destination chain.
+
+* Locking chain: The chain on which the bridge locks and unlocks assets.
+
+* Issuing chain: The chain on which the bridge mints and burns assets.
+
+* Door account: A special type of account that is used to move assets from one chain to another. The door account on a locking chain is used to put assets into trust, and the door account on an issuing chain used to issue wrapped assets. 
+
+
+## Transactions
+
+### Bridge Control: XChainModifyBridge
+
+This transaction modifies a Bridge ledger object on one of the chains that the bridge connects. You can only change the SignaturesReward or MinAccountCreateAmount values, because changing the bridge itself (either door account or either currency) would essentially render all cross-chain funds useless - you’d be better off creating another bridge instead. This transaction must be sent by the door account, and correctly signed using whatever signer list set it has.
+
+### Cross-Chain Transfer: XChainCreateClaimID
+
+This transaction checks out a cross-chain claim ID that is used for a cross-chain transfer. It is submitted on the destination chain, not the source chain. This is the first step of a cross-chain transfer of value. 
+
+A cross-chain claim ID essentially represents one cross-chain transfer of value. 
```

Comment:
I think "represents" is not quite the word here, because it could give the impression that claiming the ID performs some of the transfer. "Identifies" is perhaps too close to defining "ID" with itself, so maybe "tracks" is better? Also, I like changing the word order so that "one transfer" is together, since that's the important part.

```suggestion
A cross-chain claim ID tracks one transfer of value across chains. 
```